argocd/assets/dashboards/argo_cd_overview.json

```diff
@@ -0,0 +1,2352 @@
+{
+    "author_name": "Datadog",
+    "description": "## Argo CD Overview\n\n### Our Argo CD dashboard gives you broad visibility into your Argo CD Clusters\n\nThis dashboard provides a high-level overview of your Argo CD clusters so that you can monitor its deployments, performance and and the overall health of the cluster.\n\n### Useful Links\n\n- [Datadog Argo CD Integration Documentation](https://docs.datadoghq.com/integrations/argocd/)\n- [Argo CD Official Documentation](https://argo-cd.readthedocs.io/en/latest/)\n\nClone this template dashboard to make changes and add your own graph widgets. (cloned)",
+    "layout_type": "ordered",
+    "template_variables": [
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "host",
+            "prefix": "host"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "kube_cluster_name",
+            "prefix": "kube_cluster_name"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "kube_namespace",
+            "prefix": "kube_namespace"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "health_status",
+            "prefix": "health_status"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "repo",
+            "prefix": "repo"
+        }
+    ],
+    "title": "Argo CD Overview",
+    "widgets": [
+        {
+            "definition": {
+                "banner_img": "/static/images/logos/argocd_large.svg",
+                "layout_type": "ordered",
+                "show_title": false,
+                "title": "Overview",
+                "type": "group",
+                "widgets": [
+                    {
+                        "definition": {
+                            "background_color": "transparent",
+                            "content": "This dashboard provides a high-level overview of your Argo CD clusters so that you can monitor its deployments, performance and the overall health of the cluster\n\nUsing the dashboard, you can monitor the health of\n* Application Controller\n* API Server\n* Repository Server",
```

Comment:
```suggestion
                            "content": "This dashboard provides a high-level overview of your Argo CD clusters so that you can monitor the deployments, performance, and overall health of a cluster\n\nUsing the dashboard, you can monitor the health of\n* Application Controller\n* API Server\n* Repository Server",
```